docs/features/vision/fog_of_war_manager.md

```
@@ -0,0 +1,433 @@
+    - default value = Color(1.0, 1.0, 1.0)
+
+- texture_units_per_world_unit
+	- type int
+	- export_range
+		- min = 1
+		- max = 10000
+		- step = 1
+		- suffix = "px/length"
+    - default value = 2
+    - set 
+```gdscript
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)